domains/andros.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Andros's personal developer website",
+  "repo": "https://github.com/tanrax",
+  "owner": {
+    "username": "Andros Fenollosa Hurtado",
+    "email": ""
```

Comment:
If you don't wish to share your email address, you should add another key below with a link to any of your socials so that we can get in touch with you just in case.

Reply:
All right! I make the change. Thanks for the initiative :)